src/main/java/uk/gov/justice/probation/courtcaseservice/service/TelemetryService.java

```diff
@@ -191,6 +191,40 @@ public void trackOffenderProbationStatusUpdateEvent(OffenderEntity offenderEntit
         telemetryClient.trackEvent(TelemetryEventType.OFFENDER_PROBATION_STATUS_UPDATED.eventName, properties, Collections.emptyMap());
     }
 
+
+    public void trackOffenderProbationStatusNotUpdateEvent(OffenderEntity offenderEntity) {
```

Comment:
AC doesn't mention this. Are these helpful? There could be thousands of these in a day. @jhevans

Reply:
This is in the AC. `Probation status update which does not cause change does not trigger database call (log an event to indicate this)`